src/main/java/domain/Car.java

```diff
@@ -0,0 +1,43 @@
+package domain;
+
+import java.util.Random;
+
+
+public class Car
+{
+    private final Random random = new Random();
+    private String carName = "";
+    private int curPos = 0;
+
+    public Car(String inputCarName)
+    {
+        carName = inputCarName;
+    }
```

Comment:
[코드 컨벤션](https://github.com/woowacourse/woowacourse-docs/tree/main/styleguide/java)을 참고해서 중괄호 위치를 수정해주세요!
```suggestion
    public Car(String inputCarName) {
        carName = inputCarName;
    }
```